### PR TITLE
DOC-777: schemes and ports are ignored

### PR DESCRIPTION
**Pull request by ephox-mogran**
Related Ticket: DOC-777

Description of Changes:
* fixing incorrect details in allowed origin configuration

Pre-checks:
- [X] Branch prefixed with `feature/` or `hotfix/`
- [N/A] `_data/nav.yml` has been updated (if applicable)
- [N/A] Files has been included where required (if applicable)
- [N/A] Files removed have been deleted, not just excluded from the build (if applicable)
- [N/A] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Comment by ephox-mogran** [comment]
I have to verify these details. Will probably take a while....

File: enterprise/server/configure.md [hunk 1]
```diff
@@ -22,49 +22,49 @@ The Enhanced Media Embed server-side component require additional configuration,
 
 The `allowed-origins` element configures a list of **all** values that can be expected by the server-side components in a HTTP Origin header from your {{site.productname}} instances (see the [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Origin) for more information on the HTTP Origin header). In short, you'll need to supply a list of all the URLs that your {{site.productname}} instances will be served from without the path information.
 
+> **Note:** Unless defining regular expression based `allowed-origins`, the `scheme` (e.g. http) and the `port` (e.g. 800) are **ignored**.
```
Comment (tylerkelly13) [inline enterprise/server/configure.md:25]:
```suggestion
> **Important:** Unless defining regular expression based `allowed-origins`, the `scheme` (e.g. http) and the `port` (e.g. 800) are **ignored**. 
```

File: enterprise/server/configure.md [hunk 2]
```diff
@@ -108,14 +104,14 @@ ephox {
 
 {{site.requires_jsscwar_230v}}
 
-Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character.
+Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` is not excluded from the matching process.
```
Comment (tylerkelly13) [inline enterprise/server/configure.md:107]:
```suggestion
Regular expressions can be used alongside [wildcards](#wildcardsupport) for specifying `allowed-origins.origins`. To use a regular expression, start and end the expression with the forward-slash `'/'` character. When using regular expressions, the `scheme` and `port` are not excluded from the matching process.
```

**Review by tylerkelly13 — APPROVED** [review]
LGTM, just a couple minor suggestions.

File: enterprise/server/configure.md (hunk 2 above)
Comment (ephox-mogran) [inline enterprise/server/configure.md:107]: Not correct after testing. Revising...

Comment (ephox-mogran) [reply enterprise/server/configure.md:107]: Nope. Completely wrong. Fixing.

File: enterprise/server/configure.md (hunk 1 above)
Comment (ephox-mogran) [reply enterprise/server/configure.md]: I changed Note -> Important

File: enterprise/server/configure.md (hunk 2 above)
Comment (ephox-mogran) [reply enterprise/server/configure.md]: Entire sentence was deleted, because it was wrong.

Comment (ephox-mogran) [reply enterprise/server/configure.md]: Regular expressions have the same limitation. Only match the host section. I've updated the docs to match this.

File: enterprise/server/configure.md [hunk 3]
```diff
@@ -22,49 +22,49 @@ The Enhanced Media Embed server-side component require additional configuration,
...
-Add `http://server.example.com` to the `allowed-origins` list.
+Add `server.example.com` to the `allowed-origins` list.
 
 If users load {{site.productname}} from the following URLs:
 
 - `https://server.example.com/editor.php`
 - `http://server.example.com/subpage/editor.php`
 
-Add `http://server.example.com` and `https://server.example.com` to the `allowed-origins` list.
+Add `server.example.com` to the `allowed-origins` list because the scheme is ignored.
```
Comment (trcm) [inline enterprise/server/configure.md:41]: This wording is a little clumsy.

Comment (ephox-mogran) [reply enterprise/server/configure.md:41]: I agree. Thoughts?

Comment (tylerkelly13) [reply enterprise/server/configure.md:41]:
```suggestion
Add `server.example.com` to the `allowed-origins` list (The scheme is ignored by the service).
```
?

Comment (ephox-mogran) [reply enterprise/server/configure.md:41]: I thought about putting it in brackets as well

**Review by trcm — APPROVED** [review]

File: enterprise/server/configure.md (hunk 3 above)
Comment (ephox-mogran) [reply enterprise/server/configure.md:41]: This is pretty urgent. How much do you care?

Comment (trcm) [reply enterprise/server/configure.md:41]: You know me well enough to know my answer to that question

**Review by tylerkelly13 — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**